src/Android/Avalonia.Android/AvaloniaView.cs

```diff
@@ -66,7 +66,9 @@ public object? Content
 
         public override bool DispatchKeyEvent(KeyEvent? e)
         {
-            return _view.View.DispatchKeyEvent(e);
+            if (!base.DispatchKeyEvent(e))
+                return _view.View.DispatchKeyEvent(e);
+            return true;
```

Comment:
Shouldn't it be the other way around? Like we do on other platforms most of the time.

I.e., try to handle it on the Avalonia side, and if not handled - call base implementation.

```suggestion
            if (!_view.View.DispatchKeyEvent(e))
                return base.DispatchKeyEvent(e);
            return true;
```

Reply:
Good catch. Yep we need to switch order.

Reply:
Code Updated.